README.md

```diff
@@ -83,6 +83,8 @@ const api = hglib.createHgComponent(
 ```
 ### Related
 
+[![diagram of related tools](https://docs.google.com/drawings/d/e/2PACX-1vSCiCzfQ8FEyHPFSq7jJD6XmzC760xH1Zr4FIcCMzFmqAlrmYEBMId8gM42uz0okmvuEaxetyPPZ9VG/pub?w=960&h=720)](https://docs.google.com/drawings/d/1Xedi5ZRtbRdt2g20qpl_lWs4BMqc2DKZ2ZOoJvpHw9U/edit)
```

Comment:
This image doesn't have nearly the resolution necessary to make out what's in it.

Reply:
@pkerpedjiev : I've deleted all but the middle view (so 960x720 is sufficient, or maybe even a bit much). Also added a key for the colors. Merge if it looks good to you?